users/views.py

```diff
@@ -0,0 +1,89 @@
+import re
+import json
+import bcrypt
+import jwt
+
+from django.views import View
+from django.http  import JsonResponse
+
+from .utils       import email_validator, password_validator, phone_validator
+from my_settings  import ALGORITHM, SECRET
+
+from .models      import User
+class SignUpView(View):
```

Comment:
```suggestion
from .models      import User

class SignUpView(View):
```
모든 임포트가 끝난뒤에 클래스 선언을 시작할때에는 두칸을 띄워 주는게 pep8 컨벤션이고,  위코드에서는 한줄 띄어도 무방합니다. 
지속적인 컨벤션 가이드에 지치실 수도 있지만, 협업시에는 이러한 최소한의 규칙이 제일 중요합니다.
컨벤션을 숙지하고 그에 맞게 코딩하는 자세 또한 중요합니다 😀

Reply:
네네 컨벤션 확실히 배우겠습니다 신경써주셔서 감사합니다!